src/couch_index_server.erl

```diff
@@ -38,7 +38,6 @@
 -record(st, {root_dir}).
 
 start_link() ->
-    couch_epi:register_service(couch_index),
```

Comment:
retracted, couch_epi is going to consult sys.config and `couch_index_epi` is listed there.